README.rst

```diff
@@ -18,15 +18,33 @@ Installation
 Usage
 =====
 
-::
-
-    >>> import axelrod_fortran as axlf
-    >>> import axelrod as axl
-    >>> p1 = axlf.Player('k31r')
-    >>> p2 = axlf.Player('k33r')
-    >>> match = axl.Match((p1, p2), turns=5)
-    >>> match.play()
-    [(C, C), (C, C), (C, D), (C, D), (C, C)]
+Running a match:
+
+.. code-block:: python
+
+   >>> import axelrod_fortran as axlf
+   >>> import axelrod as axl
+   >>> p1 = axlf.Player('k31r')
+   >>> p2 = axlf.Player('k33r')
+   >>> match = axl.Match((p1, p2), turns=5)
+   >>> match.play()
+   [(C, C), (C, C), (C, D), (C, D), (C, C)]
+
+Running an instance of Axelrod's second tournament:
+
+.. code-block:: python
+
+   >>> import axelrod as axl
+   >>> from axelrod_fortran.strategies import characteristics
```

Comment:
to match up with `axelrod` (and the lines of code written directly above), can we remove these two imports and just `import axelrod_fortran ax axlf` and then we can access:

```
axlf.charactersitics
axlf.Player
```